### PR TITLE
refactor: drop django-mptt in favour of django-tree-queries

### PR DESCRIPTION
**Pull request by winged**
Django-Tree-Queries works with recursive CTEs (common table expressions) to deal with hierarchical relationships, generally achieving the same result, but without the `right`, `left` and `tree_id` fields.

Django-MPTT has been unmaintained for several years, and we're suspecting some subtle bugs in it that we can't really reproduce or verify right now. Anyway it's a good time to get rid of it.

BREAKING! This breaks if Emeis is used as a Django app directly, and depends on any of the MPTT API / features.

### CODE REVIEW

File: emeis/core/models.py [hunk 1]
```diff
@@ -168,20 +169,21 @@ class Scope(MPTTModel, UUIDModel):
...
+        # called by DRF (only by django forms), so we have to do this here
+        self.clean()
+        return super().save(*args, **kwargs)
+
     def __str__(self):
         return f"{type(self).__name__} ({self.full_name}, pk={self.pk})"
 
     class Meta:
-        ordering = ["full_name"]
+        ordering = ["name"]
```
Comment (czosel) [inline emeis/core/models.py:186]: Why this change?

Comment (czosel) [reply emeis/core/models.py:186]: Ah, because it's not supported anymore, right?

**Review by czosel — APPROVED** [review]
Nice work!

File: emeis/core/models.py (hunk 1 above)
Comment (winged) [reply emeis/core/models.py:186]: Yes and no. django-tree-queries uses it to sort siblings, and it doesn't support `.order_by()` anymore (which may be problematic in some specific use cases, but I think our frontend doesn't care / expects the output to be in tree-order anyway)

**Comment by czosel** [comment]
Once the conflicts are resolved we can merge :+1:

**Review by czosel — APPROVED** [review]
